### PR TITLE
Add is_developer_reply property to ratings API responses

### LINKED ISSUE #8993: Expose a new attribute `is_developer_reply` in Rating API results
See: https://github.com/mozilla/addons-frontend/issues/5684

---

When fetching the ratings of a given user, we receive both the ratings and replies to the reviews left by this user/developer. We have no way to determine if a given review is a review or a reply, because we don't receive the full `addon` object along with the review.

We talked about this on IRC with @eviljeff and @diox: it should be possible to add a `is_developer_reply` boolean attribute to the rating object (by checking if a FK is present but without loading it for better performances).

### PR DESCRIPTION
**Pull request by diox**
Fix #8993

### CODE REVIEW

File: docs/topics/api/overview.rst [hunk 1]
```diff
@@ -223,3 +223,4 @@ v4 API changelog
   https://github.com/mozilla/addons-server/issues/8794
 * 2018-07-18: added ``previews`` property to discovery API ``addons`` object. This change was also backported to the `v3` API. https://github.com/mozilla/addons-server/issues/8863
 * 2018-07-20: dropped ``downloads`` property from the collection add-ons results. https://github.com/mozilla/addons-server/issues/8944
+* 2018-08-09: added ``is_developer_reply`` property to ratings. This changed was also backported to the `v3` API. https://github.com/mozilla/addons-server/issues/8993
```
Comment (diox) [inline docs/topics/api/overview.rst]: Will update as it won't make the tag.

Comment (eviljeff) [reply docs/topics/api/overview.rst]: The sooner we get #8899 done the sooner we can stop this risky modification of the "frozen" v3 API 😿

File: src/olympia/ratings/serializers.py [hunk 2]
```diff
@@ -44,6 +45,9 @@ def get_addon(self, obj):
             'slug': addon.slug
         }
 
+    def get_is_developer_reply(self, obj):
```
Comment (eviljeff) [inline src/olympia/ratings/serializers.py]: observation: a property that could be on the `Rating` model itself.  Not that it'd give us anything extra, really.

Comment (diox) [reply src/olympia/ratings/serializers.py]: Yeah, I've looked through the code and we wouldn't really make use of that property anywhere. Generally we care more about the fact that a rating has a reply than the fact that it *is* a reply.

**Pull request merged**